### PR TITLE
Improved upon the Buy Gems Window for #2238

### PR DESCRIPTION
**Pull request by divyanshu013**
Added the header and close button for the Buy Gems Window.

### CODE REVIEW

File: app/templates/play/modal/buy-gems-modal.jade [hunk 1]
```diff
@@ -8,7 +9,8 @@
 
     else
       img(src="/images/pages/play/modal/buy-gems-background.png")#buy-gems-background
-
+      h1(data-i18n="buy_gems.buy_gems")
```
Comment (Imperadeiro98) [inline app/templates/play/modal/buy-gems-modal.jade]: Could you use "play.buy_gems", instead (cause there is already a "Buy Gems" string), and then remove the one you added to the en.coffee file?

**Comment by divyanshu013** [comment]
Sure @Imperadeiro98

**Comment by nwinter** [comment]
Thanks @divyanshu013!

**Comment by divyanshu013** [comment]
You're welcome @nwinter :smiley:

**Pull request merged**